### ISSUE TITLE
1.17.1 Iris world. biomes not recognized

### ISSUE THREAD

**Issue by PaintsplattersMC** [StrangeOne101/RealBiomeCondition#3]
I am running iris 1.9.4 for 1.17.1.

my mob "gnome" is valid. I can spawn it with /mm m spawn gnome
randomspawning is enabled in mm config.yml. Other randomspawns that dont use the realbiome or biometype condition do work

this randomspawn file

```random_gnome:
  Action: ADD
  MobType: gnome
  Chance: 1
  Priority: 1
  Conditions:
![2022-04-10_18 57 21](https://user-images.githubusercontent.com/83564809/162649378-d2c4321d-d06b-4611-9faf-cf465d8ce1b8.png)

  - realbiome overworld:tunmagforest true``` does not spawn gnomes in the overworld:tunmagforest biome

even with a 100% chance of spawning and no other conditions. 0 of them spawn.

**Comment by StrangeOne101** [comment]
I've also encountered this but it's only sometimes. Right now it's an issue I have no idea how to properly replicate nor fix

**Comment by PaintsplattersMC** [comment]
this is a serious issue. it defeats the entire purpose of the plugin

**Comment by StrangeOne101** [comment]
> this is a serious issue. it defeats the entire purpose of the plugin

You're right. It does kinda defeat the purpose. But I also don't know how to fix it and only found out that it happens *sometimes* after I published it to GitHub

The plugin is open source. If anyone wants to continue or better it, they're welcome to. I don't have a huge amount of time to invest into figuring out this issue, but if anyone does figure it out then I'm open to accepting PRs
